src/lib.rs

```diff
@@ -152,6 +152,18 @@ impl Arguments {
         Arguments(args)
     }
 
+    /// Returns the name of the subcommand, that is, the first positional argument.
+    #[inline(never)]
```

Comment:
Actually, does this inline do anything? I'd like to say something like "hey, even with LTO, don't bother inlining this", but I am not sure if lto actually looks at `inline(never)`?

Reply:
According to the `cargo-bloat` it adds only 260B, so we can skip the `inline` attribute completely and let Rust decide what to do.